Backoffice/Security/Authorization/Voter/AbstractNodeVoter.php

```diff
@@ -25,45 +25,50 @@
      * Vote for Read action
      * A user can read a node if it is in his perimeter
      *
-     * @param NodeInterface $node
-     * @param UserInterface $user
+     * @param NodeInterface  $node
+     * @param TokenInterface $token
      *
      * @return bool
      */
-    protected function voteForReadAction($node, UserInterface $user)
+    protected function voteForReadAction($node, TokenInterface $token)
     {
-        return $user->hasRole(ContributionRoleInterface::NODE_CONTRIBUTOR)
+        $user = $token->getUser();
```

Comment:
@alavieille : there's no more the need to create the local var $user, you can directly inject $token->getUser() in the condition